src/components/Chat/MessageWithLink.js

```diff
@@ -70,14 +70,14 @@ const styles = StyleSheet.create({
     marginTop: 5
   },
   authorName: (isMe) => ({
-    color: isMe ? COLORS.white : COLORS.blackgrey,
+    color: isMe ? COLORS.almostBlack : COLORS.gray400,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color change from `COLORS.white` to `COLORS.almostBlack` for the author's name when `isMe` is true might affect readability if the background color is dark. Please ensure that this color change doesn't negatively impact the user experience.
```diff
-    color: isMe ? COLORS.white : COLORS.blackgrey,
+    color: isMe ? COLORS.almostBlack : COLORS.gray400,
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->